care/facility/migrations/0426_alter_patientmetainfo_occupation.py

```diff
@@ -0,0 +1,54 @@
+# Generated by Django 4.2.10 on 2024-04-05 05:17
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("facility", "0425_merge_20240403_2055"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="patientmetainfo",
+            name="occupation",
+            field=models.IntegerField(
+                blank=True,
+                choices=[
+                    (1, "STUDENT"),
+                    (2, "BUSINESSMAN"),
+                    (3, "HEALTH_CARE_WORKER"),
+                    (4, "HEALTH_CARE_LAB_WORKER"),
+                    (5, "ANIMAL_HANDLER"),
+                    (6, "HEALTHCARE_PRACTITIONER"),
+                    (7, "PARADEMICS"),
+                    (8, "Business_related"),
+                    (9, "ENGINEER"),
+                    (10, "TEACHER"),
+                    (11, "Other_Professions"),
```

Comment:
Do we need both "Other_Pressions" and "OTHERS"

Reply:
@rithviknishad maybe we can consider other skilled professions

Reply:
@rithviknishad in the frontend Other professional occupations were mentioned and professional occupation is a separate type and the corresponding value is OTHER_PROFESSIONS not to be confused with OTHERS and its not the same. This is needed.

Reply:
Rename it in backend too. It could lead to confusions.

Also change to use uppercase for all